### ISSUE TITLE
[Github] Support Github Enterprise Server REST API for Audit logs

### ISSUE THREAD

**Issue by kcreddy** [elastic/integrations#6643]
Current `github.audit` datastream only supports [Github Enterprise Cloud REST API](https://docs.github.com/en/enterprise-cloud@latest/rest/orgs/orgs?apiVersion=2022-11-28#get-the-audit-log-for-an-organization)., i.e., `https://api.github.com/orgs/ORG/audit-log`  

Need to add support for [Github Enterprise Server REST API](https://github.com/elastic/sdh-beats/issues/3445#:~:text=Github%20Enterprise%20Server%20Audit%20Log%20REST%20API) to allow users to fetch audit logs across all Github enterprise plans.

Possible changes:

- API URL and parameters in `manifest.yml` and `httpjson.yml.hbs`
- Fields addition due to response schema differences (if any) in `fields.yml`
- Ingest pipeline for handling response schema differences (if any) in `ingest_pipeline/default.yml`

**Comment by diwangca** [comment]
Is there a plan when this can be done?

**Comment by haetamoudi** [comment]
According to the documentation, the github integration does now support Enterprise Server https://www.elastic.co/guide/en/integrations/current/github.html I think we can close this issue @kcreddy

**Comment by kcreddy** [comment]
Correct. The support is added in https://github.com/elastic/integrations/pull/11918
